release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
```

Comment:
```suggestion
* Use *pipeline-as-code* as an option in the *Import from Git* workflow.
```
The list item needs to have parallel syntactic structure with the other list items. My edit is one way to achieve this change, but feel free to try alternatives if my suggestion doesn't feel right to you.

Reply:
Made this change. Thanks!